src/index.js

```diff
@@ -88,6 +92,9 @@ export default React.createClass({
 
     // Add any custom class names at the end.
     className && classNames.push(className)
-    return React.createElement(tag, { ...props, className: classNames.join(' ') })
+    return React.createElement(
+      tag, { ...props, 'aria-hidden': true, className: classNames.join(' ') },
```

Comment:
Good question. I don't see much reasons a user would want it not to be `aria-hidden` though.